src/main/java/uk/gov/hmcts/darts/retention/mapper/RetentionPolicyMapper.java

```diff
@@ -13,18 +13,20 @@
 public class RetentionPolicyMapper {
     public List<GetRetentionPolicy> mapToRetentionPolicyResponse(List<RetentionPolicyTypeEntity> retentionPolicyTypeEntity) {
 
-        return retentionPolicyTypeEntity.stream().map(o -> {
-            GetRetentionPolicy getRetentionPolicy = new GetRetentionPolicy();
-            getRetentionPolicy.setPolicyEndAt(o.getPolicyEnd());
-            getRetentionPolicy.setPolicyStartAt(o.getPolicyStart());
-            getRetentionPolicy.setId(o.getId());
-            getRetentionPolicy.setDisplayName(o.getDisplayName());
-            getRetentionPolicy.setName(o.getPolicyName());
-            getRetentionPolicy.setDescription(o.getDescription());
-            getRetentionPolicy.setFixedPolicyKey(o.getFixedPolicyKey());
-            getRetentionPolicy.setDuration(o.getDuration());
+        return retentionPolicyTypeEntity.stream().map(this::mapRetentionPolicy).toList();
+    }
+
+    public GetRetentionPolicy mapRetentionPolicy(RetentionPolicyTypeEntity o) {
```

Comment:
Generally best to avoid single-character variable names as it hurts readability. Can `o` this be renamed to something more meaningful, e.g. `entity`?

Reply:
will rename.